tests/ballerina-integration-test/src/test/java/org/ballerinalang/test/jms/util/JmsClientHandler.java

```diff
@@ -28,10 +28,10 @@
 /**
  * Client handler class to handle JMS client related operations.
  */
-public class JMSClientHandler {
+public class JmsClientHandler {
```

Comment:
Just wondering, why are we replacing JMS with Jms? This stands for, Java Message Service Client Handler right?

Reply:
It is to have the same convention in all places. We have classes named as Http* and not HTTP*. One other concern is having capital letters in classes make them harder to find in IDEs.